.vscode/settings.json

```diff
@@ -36,12 +36,6 @@
   "files.eol": "\n",
```

Comment:
are these changes somehow related?

Reply:
vscode sometimes changes these files. I think it happens when I move to different machines